app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -260,11 +260,11 @@ def display_phone_to_deliver_to
 
   def reenter_phone_number_path
```

Comment:
reverted this change because in 6caed3edb18a6af1cba803f2bb9a4c6d9a96da10 we moved away from `Link` presenter in favor of `view_context` so now we are passing these as paths instead of strings.